EduVPN-multiOS/ImageLoading/ImageLoader.swift

```diff
@@ -33,12 +33,12 @@ public class ImageLoader {
         if let data = cache.cachedResponse(for: request)?.data, let image = CrossPlatformImage(data: data) {
             target.image = image
         } else {
-            let dataTask = URLSession.shared.dataTask(with: sourceUrl, completionHandler: { (data, response, error) in
+            let dataTask = URLSession.shared.dataTask(with: sourceUrl, completionHandler: { (data, response, _) in
                 if let data = data, let response = response, ((response as? HTTPURLResponse)?.statusCode ?? 500) < 300, let image = CrossPlatformImage(data: data) {
                     let cachedData = CachedURLResponse(response: response, data: data)
                     cache.storeCachedResponse(cachedData, for: request)
                     DispatchQueue.main.async {
-                        if let _ = dataTasks.removeValue(forKey: target) {
+                        if dataTasks.removeValue(forKey: target) != nil {
```

Comment:
This is another swiftlint rule that puzzles me. The let _ seems just as readable to me.